README.md

```diff
@@ -353,25 +354,42 @@ The timestamps are return as strings\
 Status can be one of ("pending" | "accepted" | "loaned" | "returned" | "canceled" | "declined")
 
 ```json
-{
-  "id": "123",
-  "owner": {
-    "user object": "See User Object"
-  },
-  "owner_contact": "name@domain.com",
-  "requester": {
-    "user object": "See User Object"
-  },
-  "requester_contact": "name@domain.com",
-  "book": {
-    "book object": "See Book Object (but without the loan property)"
-  },
-  "request_date": "2020-04-16T15:38:49.000Z",
-  "accept_date": "2020-04-16T15:38:49.000Z",
-  "loan_start_date": "2020-04-16T15:38:49.000Z",
-  "loan_end_date": "2020-04-16T15:38:49.000Z",
-  "status": "pending"
-}
+[
+  {
+    "id": "123",
+    "requester_id": "123",
+    "book_id": "123",
+    "request_date": "2020-04-16T22:38:49.000Z",
+    "accept_date": "2020-04-16T22:38:49.000Z",
+    "loan_start_date": "2020-04-16T22:38:49.000Z",
+    "loan_end_date": "2020-04-16T22:38:49.000Z",
+    "status": "One of five status enums",
+    "book": {
+      "id": "123",
+      "user_id": "456",
+      "title": "Up to 255 chars",
+      "author": "Up to 255 chars",
+      "isbn": "101-0-10-101010-1",
+      "visibility": "One of three book visibility enums",
+      "summary": "Up to 255 chars",
+      "categories": ["Example", "AnotherExample"]
+    },
+    "owner": {
+      "id": "456",
+      "name": "Up to 255 chars",
+      "status": "One of five status enums (or null)",
+      "picture": "https://lh3.googleusercontent.com/a-/AOh14GjqisPCQh-64TyhlUaN39cQK6z0elofGrDipo8W=s96-c",
+      "email": "someone@example.com"
+    },
+    "requester": {
+      "id": "123",
+      "name": "Up to 255 chars",
+      "status": "requested",
+      "picture": "https://lh3.googleusercontent.com/a-/AOh14GjvDknrAwss4wed2u_t8oyI7o19lgnRcI9pkwxVkA=s96-c",
+      "email": "someone@example.com"
```

Comment:
The owner and requester objects should just say "See User Object" so that any future changes to the user object don't require it to be updated in more than one place